infra/scripts/local-ci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configure a vCenter cluster with vSAN datastore, DVS and DVPGs
+
+# export GOVC_INSECURE=1
+# export GOVC_USERNAME=${GOVC_USERNAME:-"Administrator@vsphere.local"}
+# if [ -z "$GOVC_PASSWORD" ] ; then
+#   # extract password from $GOVC_URL
+#   GOVC_PASSWORD=$(govc env GOVC_PASSWORD)
+# fi
+
+# usage() {
+#   echo "Usage: $0 [-d DATACENTER] [-c CLUSTER] VCSA_IP ESX_IP..." 1>&2
+#   exit 1
+# }
+
+username='nloc'
+secretfile="$1"
+odir="ci-results"
+
+mkdir -p $odir
+
+echo "Running tests"
+github_api_key="$(grep 'GITHUB_AUTOMATION_API_KEY' ${secretfile} | awk '{ print $2 }')"
+test_url="$(grep 'TEST_URL_ARRAY' ${secretfile} | awk '{ print $2 }')"
```

Comment:
For all bar the github token you may well be able to use the output from `govc env` - we heavily leverage those environment variables already so likely their set.
They are being used in #7164 and the current `bash-helpers.sh` script sets them as part of the profiles.